frontend/javascript/basket.js

```diff
@@ -0,0 +1,86 @@
+function removeCartItem() {
+    let cartRemove_btns = document.querySelectorAll('.cart-remove');
+
+    cartRemove_btns.forEach((btn) => {
+        btn.addEventListener('click', function () {
+            btn.parentElement.remove();
+            updateTotal();
+        });
+    });
+}
+
+removeCartItem();
+
+function updateTotal() {
+    let cartBoxes = document.querySelectorAll('.cart-box');
+    let totalPrice = document.querySelector('.total__price');
+    let total = 0;
+
+    cartBoxes.forEach((cartBox) => {
+        let priceElement = cartBox.querySelector('.cart-box__product_price');
+        let price = parseFloat(priceElement.innerHTML.replace('£', ''));
+        let quantity = cartBox.querySelector('.cart-box__product-quantity').innerHTML;
+        total += price * quantity;
+    });
+
+    total = total.toFixed(2);
+
+    totalPrice.innerHTML = '£' + total;
+}
+
+function changeQuantity() {
+    let cartBoxes = document.querySelectorAll('.cart-box');
+    let decreaseQtyImg = document.querySelector('.cart-box-btn__disabled ');
+    cartBoxes.forEach((cartBox) => {
+        let currentQuantityElement = document.querySelector('.cart-box__product-quantity');
+        let currentQuantity = Number(
+            document.querySelector('.cart-box__product-quantity').innerHTML
+        );
+
+        let increaseInQuantitybtn = document.querySelector('.cart-box_quantity-increase');
+        increaseInQuantitybtn.addEventListener('click', function () {
```

Comment:
To decrease the length of the `changeQuantity` function and possibly improve readability, could the callbacks of the event listeners be extracted to separate (named) functions?

Reply:
extracted callbacks into separate functions